pages/index.vue

```diff
@@ -1,24 +1,18 @@
 <template>
-  <div v-if="!mapStore.selectedLocation">
+  <div>
     <h1>Landslide Risk for Alaskan Communities</h1>
```

Comment:
I guess we actually may want this to be "...for Prince of Wales Island" but we'll let them propose that.